Makefile

```diff
@@ -63,6 +63,7 @@ vendor-update:
 
 vendor:
 	go mod vendor
+	sed -i '1 i\// +build skipcompile\n' vendor/kubevirt.io/client-go/kubecli/kubevirt_test_utils.go
```

Comment:
We should let folks know on Stork dev channel that go mod vendor will not work on Stork repo anymore. When will need to do make vendor. Else many people will end up wasting time to figure this out.

Reply:
Agree, we always recommend using the `make ....` command to do anything, and hence everything is in the Makefile. We can broadcast this on the stork channel.